bundles/org.eclipse.swt/Eclipse SWT Browser/common/org/eclipse/swt/browser/WebBrowser.java

```diff
@@ -724,6 +726,10 @@ public void setBrowser (Browser browser) {
 	this.browser = browser;
 }
 
+public URI getBaseUri() {
```

Comment:
Can this be package protected, since the only intended consumer (`Browser´) is in the same package? Please also improve the name according to field name (see comment on the field name for that).

In addition, please check whether this method can be used by the IE and WebKit implementations, which currently have their own `ABOUT_BLANK` field, which might (at least at some places) be reasonably replaced by this method.

Reply:
Made it package protected but I think refactoring of the browser specific usage can be done in a follow up PR?

Reply:
That's also fine. But then it should be done directly after merging this change.